### PR TITLE
Add accessibility audit helper

### PR DESCRIPTION
**Pull request by kevinsawicki**
Adds support for running an accessibility audit using https://github.com/GoogleChrome/accessibility-developer-tools

New API is `app.client.auditAccessibility()` and it resolves to an `audit` object with details about any failures.

/cc @jlord @zeke

### CODE REVIEW

File: test/fixtures/accessible/main.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+var BrowserWindow = require('electron').BrowserWindow
+
+var mainWindow = null
+
+app.on('ready', function () {
+  mainWindow = new BrowserWindow({
+    center: true,
+    width: 800,
+    height: 400,
+    minHeight: 100,
```
Comment (zeke) [inline test/fixtures/accessible/main.js]: what are `minHeight` and `minWidth` needed for?

Comment (kevinsawicki) [inline test/fixtures/accessible/main.js]: I copied another fixtures folder that had these set, they aren't explicitly needed by any of the new accessibility specs.

**Comment by zeke** [comment]
I wonder if you could just `npm install accessibility-developer-tools` instead of manually copying the bundled source...

**Comment by kevinsawicki** [comment]
> I wonder if you could just npm install accessibility-developer-tools instead of manually copying the bundled source...

You can't, it isn't require-able.

**Comment by zeke** [comment]
> it isn't require-able.

I took at stab at making it require-able last night. Got close but still a few hiccups. Also asked for help at https://github.com/GoogleChrome/accessibility-developer-tools/issues/291

Obviously this is just a nice-to-have. In the meantime, I say :shipit:

**Comment by kevinsawicki** [comment]
> I took at stab at making it require-able last night. Got close but still a few hiccups.

Yeah, it would be great to switch it to an npm dependency eventually if upstream addresses this issue.

**Comment by dylanb** [comment]
If you want an accessibility module that supports require (UMD) and has a bigger and more robust set of rules, take a look at axe-core https://www.npmjs.com/package/axe-core

**Pull request merged**